docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,135 @@
+- [Reachable](/semgrep-supply-chain/triage-and-remediation#filters) - if the finding is from Semgrep Supply Chain.
+- [Valid](/semgrep-secrets/getting-started#validation) - if the finding is from Semgrep Secrets.
+
+## Production backlog
+
+This pane displays analytics related to findings detected in your primary or default branch. This typically means that the finding, usually a security issue, has made it to production environments.
+
+### Key metrics
+
+| Key metrics    | Description |
+| -------------- | ------ |
+| Total new      | |
+| Total fixed    | |
+| Total ignored  | |
+| Total net new  | |
+
+### Charts
+
+| Chart | Description |
+| -------  | ------ |
+| Open backlog         | Open findings over the chosen time period. This tracks the total findings from each scan and displays them. Lower values are better. |
+| Backlog activity | Displays the number of new, net new, fixed, and ignored findings. A greater **Fixed** value is better. |
+
+## Secure guardrails
+
+This provides an overview of how secure guardrails in **PR or MR comments** are used in your organization. Other guardrail interfaces, such as the IDE or `pre-commit`, are not counted in this section.
+
+![Secure guardrails pane](/img/dashboard-guardrails.png)
+_**Figure**. Secure guardrails pane._
+
+### Key metrics
+
+<!-- vale off -->
+| Key metrics    | Description |
+| -------------- | ------ |
+| Findings shown to devs      | Percentage of findings shown to developers in PR or MR comments against the total findings count. An upward or stable trend is better. |
+| Findings fixed before backlog      | Displays the percentage of findings that were fixed before they could be detected in a default branch or production backlog against the total findings count in the specified time period. An upward or stable trend is better. |
+
+<!-- vale on -->
+
+### Charts
+
+| Chart | Description |
+| -------  | ------ |
+| Secure guardrails adoption  | Percent of new findings shown to developers over the specified time period. An upward or stable trend is better. |
+| Guardrails activity | This chart displays a breakdown of the status of findings shown to developers; whether they were ignored, fixed, or remained open. A greater **Fixed** value is better. |
+
+## Most findings by project
+
+A table providing a breakdown of findings for each project, grouped by product or severity. Lower values are better.
+
+![Findings by project pane.](/img/dashboard-findings-by-project.png#sm-width-noborder)
+_**Figure**. Findings by project; group them by product or severity._
+
+## Mean time to remediate
+
+<!-- check if title changes -->
+
+![Mean time to remediate a finding.]/img/dashboard-mean-time-remediate.png)
+_**Figure**. Mean time to remediate a finding._
+
+A chart displaying the mean time to remediate a finding over the specified time period. Lower is better.
+
+For a finding to be remediated, it must have any of the following statuses:
+
+- Fixed
+- Ignored
```

Comment:
Not sure sure about Ignored, Fixing, and Reviewing triage statuses counting as "remediation."

Reply:
Fixing / Reviewing - considered open.